sklearn/decomposition/_lda.py

```diff
@@ -399,7 +404,7 @@ def _init_latent_vars(self, n_features):
         # In the literature, this is called `lambda`
         self.components_ = self.random_state_.gamma(
             init_gamma, init_var, (self.n_components, n_features)
-        )
+        ).astype(dtype)
```

Comment:
Similar comment to the other PRs and https://github.com/scikit-learn/scikit-learn/pull/22113/files#r777245849:

```suggestion
        ).astype(dtype, copy=False)
```

Reply:
Changed.
Thank you.